source/administration/config-settings.rst

```diff
@@ -391,14 +370,44 @@ Restrict the permission level required to delete private groups. Deleted groups
 | This feature's ``config.json`` setting is ``"RestrictPrivateChannelDeletion": "all"`` with options ``all``, ``team_admin`` and ``system_admin`` for above settings respectively.   |
 +------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Allow which users to delete messages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Restrict the permission level required to delete messages.
+
+**Message authors can delete their own messages, and Administrators can delete any message**: Allow authors to delete their own messages, and allow Administrators to delete any message.
+
+**Team Admins and System Admins**: Allow only Team Admins and System Admins to delete messages.
```

Comment:
Same comment as above `...delete message in channels of which they are a member`